cmd/gateway/manta/gateway-manta.go

```diff
@@ -234,6 +246,7 @@ func (g *Manta) Production() bool {
 type tritonObjects struct {
 	minio.GatewayUnsupported
 	client *storage.StorageClient
+  ctx context.Context
```

Comment:
is it necessary to hold context in this struct? AFAIK, golang context is request scoped - so the context can be passed by each request rather than held here.

Reply:
:+1: you are right @poornas this is an incorrect addition.